Makefile

```diff
@@ -101,6 +105,10 @@ dist/flanneld-$(ARCH):
 		mv dist/flanneld dist/flanneld-$(ARCH) && \
 		file dist/flanneld-$(ARCH)'
 
+## Busybox images are missing pthread. Pull it out of the kube-cross image
+dist/libpthread.so.0-$(ARCH):
+	docker run -ti --rm -v `pwd`:/host gcr.io/google_containers/kube-cross:$(KUBE_CROSS_TAG) cp /lib/$(LIB_DIR)/libpthread.so.0 /host/dist/libpthread.so.0-$(ARCH)
```

Comment:
Is there a way to not pull out a lib out of some other container? Can we work to get this included into busybox-glibc image?